README.md

```diff
@@ -57,6 +57,22 @@ Observe Basilisk logs
 multitail 99*.log
 ```
 
+### Testing Runtime
+
+There is also an option to run the testing runtime with less restrictive settings to facilitate testing of new features.
+The following command starts a dev chain, and the testing runtime is used as a runtime for our node.
+```bash
+./target/release/hydra-dx --dev --runtime=testing
+```
+The testing runtime currently supports only two chain specifications: _dev_ and _local_ testnet.
+Both runtimes store blockchain data in the same directories( e.g. the _dev_ directory is shared for both runtimes 
+started with the `--dev` parameter. That's why it is important to purge chain data when switching to different runtime( note: `--runtime` parameter can't be used when purging chain data)
+
+In the case of starting a testnet using the `polkadot-launch` tool, 
+we don't have an option to communicate to its internal commands that we would like to use the testing runtime.
+To overcome this limitation, rename the binary so it starts with the `testing` prefix, e.g. `testing-basilisk`.
+Such a binary always uses the testing runtime, even if the `--runtime testing` option is not specified.
```

Comment:
I feel that there must be a different solution to this.  I personally don't like it - I can already see the questions "why it does not work? -> because you forgot to rename the binary".

Which does not make much sense - it should not be dependent on binary name.

I vaguely remember your explanation why it is necessary. But it still quite clear according to this description.

Perhaps - having possibility to compile in only selected runtime would help ? via a specific feature?

Because we currently compile all runtimes regardless - so i would really like an option to select which runtime i want to use. And that might also resolve this issue ? 
By default, it could compile all runtimes, but option to select a specific one would be nice. 
Just an idea.

Reply:
The issue is that when `polkadot-launch` starts a relaychain, it calls `basilisk export-genesis-state` and `basilisk export-genesis-wasm` commands and we don't have an option to tell `polkadot-launch` to add `--runtime testing` argument to those commands. As a result, parathreads are registered with incorrect genesis-state and wasm runtime validation function (for the main runtime) but it tries to spawn parachains with testing runtime.

Reply:
@Roznovjak Can you try to look at polkadot-launch how much effort will be to add passthrough for flags?